### DISCUSSION TITLE
How to restore only the diff between current state and a particular backup?

### DISCUSSION THREAD

**Discussion by VictorVow** [borgbase/vorta#2053] [General]
I've had a hard drive problem but have been able to restore to the state of my PC from two weeks ago using a local backup. 

I've got a more recent backup from 1 day ago available on a cloud Vorta repo. I dont want to download everything from the cloud backup as my internet is too slow.

How can I selectively restore only the changes that have happened since two weeks ago that are in this cloud Vorta backup?

**Comment by jeop10** [comment]
Hello!

Disclaimer: I am new and im just trying to learn Vorta as well

I think that what you are asking can be achieved by mounting that archive into an empty directory and then copying the files that you want into the folders that you want.

Or if you already know the files that you want you can use the "extract" functionality. 

Here is a link to the official docs where they cover the two different approaches mount vs extract: https://vorta.borgbase.com/usage/restore/

I think the performance of this process will depend on the size repo and the speed of your internet connection.

Regards
